test/checkpoint/test_checkpoint.c

```diff
@@ -277,7 +283,7 @@ onint(int signo)
 {
 	WT_UNUSED(signo);
 
-	(void)cleanup();
+	cleanup();
```

Comment:
The cast to void is necessary here to avoid compiler warnings. Any time the return value of a function that doesn't return void is ignored, we cast it to void - it's a way of acknowledging that we are deliberately ignoring the error return.